src/convertToObject.js

```diff
@@ -6,7 +6,21 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const stylesArray = sourceString.trim().split(';');
+
+  const stylesObject = {};
+
+  stylesArray.forEach((style) => {
+    if (style.trim() === '') {
+      return;
+    }
+
+    const [property, value] = style.split(':');
```

Comment:
Ensure to handle cases where the CSS declaration might not contain a colon or have more than one colon, which could result in an undefined value or incorrect splitting.